features/mbedtls/targets/TARGET_STM/aes_alt.c

```diff
@@ -182,21 +247,41 @@ int mbedtls_aes_crypt_cbc( mbedtls_aes_context *ctx,
     int status = 0;
     if( length % 16 )
         return( MBEDTLS_ERR_AES_INVALID_INPUT_LENGTH );
+    ctx->hcryp_aes.Init.pInitVect = &iv[0];
+    status |= st_cbc_restore_context(ctx);
```

Comment:
One of my review comments is shown as outdated, reposting it for visibility:

I think we should check the return value here and return on error, because otherwise it is theoretically possible for a compromised application to use another applications key/context if it gets the timing right.

Reply:
Correct. 
I have 6 times `status |=...`
Do you think I should replace each of them by 
`if (... ) return 1;`
?

Reply:
Yes, I think that would be good practice and a more defensive coding!